.vscode/settings.json

```diff
@@ -9,6 +9,8 @@
     "files.associations": {
         "*.bs.include": "bikeshed",
     },
+    "files.eol": "\n",
+    "files.trimTrailingWhitespace": true,
```

Comment:
This is much less important but I'd like if we also set `files.insertFinalNewline` for consistency across contributors. (case in point, `.gitattributes` doesn't have one 🙂 )

Reply:
@kainino0x I think that's a great one! I actually didn't know about it, despite pondering if there's a better way to streamline that 😆. I used following script (based on check-nfkc) to check things besides including the settings:

```
#!/usr/bin/env python3

import subprocess
import sys
import os

def normalize_file_contents(contents: str) -> str:
    lines = contents.splitlines()
    lines = [line.rstrip() for line in lines]
    normalized = "\n".join(lines)
    if not normalized.endswith("\n"):
        normalized += "\n"
    return normalized

def is_different(original: str, normalized: str) -> bool:
    return original != normalized

def main():
    fix = "--fix" in sys.argv
    files = subprocess.run(
        ["git", "ls-files"], check=True, capture_output=True, text=True
    ).stdout.splitlines()
    wrong_files = []
    for file in files:
        if os.path.isdir(file) or ".svg" in file or ".json" in file:
            continue
        with open(file, "rb") as f:
            raw = f.read()
        try:
            original_contents = raw.decode("utf-8")
        except UnicodeDecodeError:
            continue

        normalized = normalize_file_contents(original_contents)

        if is_different(original_contents, normalized):
            if fix:
                # Rewrite the file
                with open(file, "w", encoding="utf-8", newline="\n") as f:
                    f.write(normalized)
            else:
                wrong_files.append(file)

    if len(wrong_files) > 0:
        print("The following files have inconsistent line endings:")
        for file in wrong_files:
            print(file)
        sys.exit(1)

if __name__ == "__main__":
    main()

```